Nos/Models/Event+CoreDataClass.swift

```diff
@@ -728,11 +728,18 @@ public class Event: NosManagedObject {
     }
     
     var isReply: Bool {
-        rootNote() != nil
+        rootNote() != nil || referencedNote() != nil
     }
     
     /// Returns the event this note is directly replying to, or nil if there isn't one.
     func referencedNote() -> Event? {
+        if let rootReference = eventReferences?.first(where: {
+            ($0 as? EventReference)?.marker ?? "" == "reply"
```

Comment:
I think it would be better to make an enum inside Event with some values the marker can take, like root and reply, and use that value here and in other queries we make (they are easily found by searching marker in the project). That way we control the values somewhere and make it easy for other developers to find out what marker usually means.

Reply:
Good idea. I cleaned this up.